src/index.css

```diff
@@ -1,8 +1,76 @@
-body, h1, h2, h3, h4, h5, h6, p, ol, ul {
-    margin: 0;
-    padding: 0;
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ol,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Open Sans", sans-serif; /* Add your preferred font-family */
+}
+
+.body-post {
+  text-align: center;
+}
+
+.slider-container {
```

Comment:
is this for the review component?

Let's not implement the css here.

should be in the `src/components/Body` or anywhere the slider component presents

Reply:
udah saya pindahkan mbak